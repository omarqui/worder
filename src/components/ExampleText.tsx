import React, { useState, FunctionComponent }  from 'react';
import "../App.css";

type ExampleTextProps = {
    value: string
}

export const ExampleText = ({value}: ExampleTextProps)=>{
    return (
        <div>
            <span className="enfasis">Example:</span>
            <span className="example">"{value}"</span>
        </div>
    )
}
