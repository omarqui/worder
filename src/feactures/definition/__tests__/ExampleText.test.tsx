import React from "react";
import { render } from "@testing-library/react";
import { ExampleText } from "../ExampleText";

it("should ExampleText match with snapshot", () => {
    const { container } = render(<ExampleText value="hola" />)
    expect(container).toMatchSnapshot()
})