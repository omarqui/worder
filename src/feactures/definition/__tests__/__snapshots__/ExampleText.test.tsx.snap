// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should ExampleText match with snapshot 1`] = `
<div>
  <div>
    <span
      class="enfasis"
    >
      Example:
    </span>
    <span
      class="example"
    >
      "
      hola
      "
    </span>
  </div>
</div>
`;
